CRM/Upgrade/Incremental/sql/5.26.alpha1.mysql.tpl

```diff
@@ -1 +1,4 @@
 {* file to handle db changes in 5.26.alpha1 during upgrade *}
+
+UPDATE civicrm_contact SET is_deceased = 0 WHERE is_deceased IS NULL;
+ALTER TABLE civicrm_contact MODIFY COLUMN is_deceased TINYINT NOT NULL;
```

Comment:
Should add DEFAULT 0 on the end, i.e. `ALTER TABLE civicrm_contact MODIFY COLUMN is_deceased TINYINT NOT NULL DEFAULT 0;`, otherwise it removes the default.

Reply:
good catch - thank you!!